cloud-service-providers/azure/aks/prerequisites/README.md

```diff
@@ -0,0 +1,74 @@
+# Pre-requirement
+
+The keys to create AKS for NIM is to create proper GPU nodepool.  The proper GPU nodepool should have up to date driver.  This is only achievable via a preview cli extension.
```

Comment:
Can we reword this to say that the driver must meet the NIM minimal requirements and to say why the nodepool matters?